playground/mocks/data/idp/idx/identify-with-username-password.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
We can use https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/identify-with-password.json, it is the same (I think), so no need to create a new file